### ISSUE TITLE
Compound var declarations

### ISSUE THREAD

**Issue by cbarrick** [atom/language-go#66]
After the update to Atom v1.1.0, comments and other lexical elements within var blocks don't highlight properly. This is independent of syntax-theme.

![var block](https://cloud.githubusercontent.com/assets/271982/10860964/f16e9930-7f47-11e5-8f71-38b6de387d9e.png)

Const blocks still highlight correctly, so I've included such a screenshot as a comparison:

![const block](https://cloud.githubusercontent.com/assets/271982/10861004/fa4c31b0-7f48-11e5-9538-46016bd60f3a.png)

**Comment by 50Wliu** [comment]
Hi there @cbarrick!  Can you please move this to the [language-go](https://github.com/atom/language-go) repository?  Thanks!

/cc @svanharmelen

**Comment by MaximSokolov** [comment]
Here is rule that causes this: [go.cson#L472-L475](https://github.com/atom/language-go/blob/86aeed0221b0559cd37ffffef59cd269f2f60446/grammars/go.cson#L472-L475)

```coffee
# Rule is part of `variables` repository
{
  'match': '[a-zA-Z_]\\w*'
  'name': 'variable.other.declaration.go'
}
```

Code:
```go
var (
	// Count of the number of fitness evaluations
	count struct {
		sync.Mutex
		n int
	}
)
```
"One Dark" syntax theme on the screenshot

**Comment by joefitzgerald** [comment]
@50Wliu this is the language-go repository. Not sure why you're asking for it to be moved here, when it's already here?

**Comment by 50Wliu** [comment]
@joefitzgerald Sorry! :see_no_evil: Sometimes it happens when I've just been going through all the atom/atom issues and then click on one from a different repo.

**Comment by michaeltrobinson** [comment]
+1 - I also just ran into this after upgrading to Atom 1.1

**Comment by fortytw2** [comment]
Same problem here after upgrading to Atom 1.1, comments aren't highlighting properly -> 
![screen shot 2015-11-04 at 14 58 36](https://cloud.githubusercontent.com/assets/10286538/10939823/9d5b890a-8304-11e5-8fa2-6e070507c1a5.png)

**Comment by svanharmelen** [comment]
This one is fixed by PR #67
